src/js/tech/html5.js

```diff
@@ -1322,8 +1322,7 @@ Html5.disposeMediaElement = function(el) {
     el.removeChild(el.firstChild);
   }
 
-  // remove any src reference. not setting `src=''` because that causes a warning
-  // in firefox
+  el.src = '';
```

Comment:
seems like firefox no longer shows a warning here.

Should we add this to `resetMediaElement` below as well? Line 1356